lib/Runtime/Base/jitprofiling.cpp

```diff
@@ -0,0 +1,266 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft Corporation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#include "RuntimeBasePch.h"
+
+#include "ittnotify_config.h"
+
+#if ITT_PLATFORM==ITT_PLATFORM_WIN
+#include <windows.h>
+#pragma optimize("", off)
```

Comment:
Good finding, thanks! It maybe be a workaround for some compiler issues and it's no longer relevant. We have tested by removing this line on VTune code base as well as ChakraCore with and without VTune, no negative impact has been found. So will remove this line: #pragma optimize("", off) and update the PR.